lib/fibonacci.rb

```diff
@@ -4,6 +4,24 @@
 # e.g. 1st fibonacci number is 1
 # ....
 # e.g. 6th fibonacci number is 8
+# fibonacci(3).must_equal 2
+
+# Time Complexity: O(n)
```

Comment:
I'd recommend practicing elaborating the reasoning for your time and space complexity.